scripts/module_list/module_overview.py

```diff
@@ -0,0 +1,175 @@
+import os
+import subprocess
+from typing import Union, Tuple
+from mdutils.mdutils import MdUtils
+import numpy as np
+
+
+# --------------------------------------------------------------------------------------------------------
+# Module bash API
+# --------------------------------------------------------------------------------------------------------
+
+def module(*args, filter_fn=lambda x: x) -> np.ndarray:
+    """
+    API to call the module command.
+
+    @param args: Extra arguments for the module command.
+    @param filter_fn: Filter function on the ouput.
+    @return:
+    """
+    lmod = os.getenv('LMOD_CMD')
+    proc = subprocess.run(
+        [lmod, "python", "-t"] + list(args),
+        encoding="utf-8",
+        stderr=subprocess.PIPE,
+        stdout=subprocess.PIPE
+    )
+    exec(proc.stdout)
+    return filter_fn(np.array(proc.stderr.split()))
```

Comment:
Maybe add a comment to explain what is contained in `stdout` (Python statement to change current environment) and `stderr` (actual output produced by module command, in machine-readable format because we use `--terse`)